osier/equations.py

```diff
@@ -143,32 +247,34 @@ def annual_co2(technology_list, solved_dispatch_model, emission='co2'):
     solved_dispatch_model : :class:`osier.DispatchModel`
         A _solved_ dispatch model (i.e. with model results and objective
         attributes).
-    
+
     Returns
     -------
     emissions_total : float
         The total emissions of the technology set.
     """
-    
+
     dispatch_techs = get_dispatchable_techs(technology_list)
     non_dispatch_techs = get_nondispatchable_techs(technology_list)
     column_names = get_tech_names(technology_list)
     dispatch_results = solved_dispatch_model.results
 
-    emissions = np.array([getattr(t, emission) 
-                         for t in dispatch_techs+non_dispatch_techs 
-                         if hasattr(t,emission)])
+    emissions = np.array([getattr(t, emission)
+                         for t in dispatch_techs + non_dispatch_techs
+                         if hasattr(t, emission)])
 
-    emissions_total = np.dot(emissions, dispatch_results[column_names].values.T).sum()
+    emissions_total = np.dot(
+        emissions,
+        dispatch_results[column_names].values.T).sum()
```

Comment:
This is really similar to what you do in `per_unit_energy`. Would it make sense to create a method to do this?

Reply:
This is a good point, now that I have the generic functions, I should be calling those for each unique objective.

Reply:
I'm realizing that might not have been what you meant. Why would I have an emissions calculator when `per_unit_energy` can also do that? Usability. In order for users to use `per_unit_energy` they have to do

```py
import functools
objectives = [functools.partial(per_unit_energy, attribute='co2_rate')]
```
While straightforward, it's not as readable as 

```py
objectives = [annual_emission]
```

I'll keep adding functions to `equations.py` because it's always clearer to have a specific function. The generic version adds functionality but sacrifices usability.